cmd/cmd.go

```diff
@@ -1684,6 +1749,8 @@ func NewCLI() *cobra.Command {
 	runCmd.Flags().String("think", "", "Enable thinking mode: true/false or high/medium/low for supported models")
 	runCmd.Flags().Lookup("think").NoOptDefVal = "true"
 	runCmd.Flags().Bool("hidethinking", false, "Hide thinking output (if provided)")
+	runCmd.Flags().Bool("truncate", false, "For embedding models: truncate inputs exceeding context length (default: true). Set --truncate=false to error instead")
```

Comment:
```suggestion
	runCmd.Flags().Bool("truncate", false, "Truncate inputs exceeding context length (default: true). Set --truncate=false to error instead (for embedding models only)")
```

I'd make the description format more consistent with the one below